Simulator_engine/Farm.py

```diff
@@ -367,24 +370,49 @@ def update_weigh(self,prod_time):
         return self.W
 
     def updateCF(self):
-        #  if there is no data for cleaner fish dont update it
+        #  if there is no data for cleaner fish don't update it
         self.cleaner_fish = self.cleaner_count_update(self.time)
 
         #  How many lice do cleaner fish eat in one delta time step (0.05 per day)
-        self.cleaner_death = self.cleaner_fish * self.cleanEff *self.delta_time 
+        self.cleaner_death = self.cleaner_fish * self.cleanEff * self.delta_time
 
         sum_mobile_lice = np.sum(
             [
                 np.sum(self.get_fordeiling()[2:6]), 
                 np.sum(self.get_fordeiling()[8:])
             ]
         )
-
-        if self.cleaner_death <=0 or sum_mobile_lice==0 or np.isnan(self.cleaner_death):
+        print(sum_mobile_lice, self.CF_lice_min, sum_mobile_lice / self.fish_count, self.cleaner_death,(sum_mobile_lice/self.fish_count) >= self.CF_lice_min)
```

Comment:
change `>=` to `>`, so that if `CF_lice_min==0` this code runes for the first couple of cycles